src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+
```

Comment:
there should not be any empty lines after class declaration
same for all classes